core/templates/pages/creator-dashboard-page/suggestion-modal-for-creator-view/suggestion-modal-for-creator-view.directive.html

```diff
@@ -59,4 +59,11 @@ <h3>Review Suggestion</h3>
   .oppia-edit-mode .oppia-review-message {
     margin-top: 10px;
   }
+  .oppia-edit-mode .oppia-state-not-exist-text {
```

Comment:
This is quite a weird prefix. I understand that it was already in this file but it doesn't seem to be related to the directive in question, and especially those classes that you've added. Could you please fix this?